docs/src/main/sphinx/language/types.rst

```diff
@@ -239,11 +239,14 @@ The following examples illustrate the behavior::
 Instant in time that includes the date and time of day with ``P`` digits of
 precision for the fraction of seconds and with a time zone. Values of this
 type are rendered using the time zone from the value.
-Time zones are expressed as the numeric UTC offset value::
+Time zones are expressed as the numeric UTC offset value or :doc:`IANA time zone
```

Comment:
Should this be plain link instead of `doc`? Sphinx is complaining.

Reply:
Yes, syntax is

```
`text <link>`_
```

Reply:
Whoops. Fixing